railties/lib/rails/tasks/statistics.rake

```diff
@@ -1,7 +1,18 @@
 # While global constants are bad, many 3rd party tools depend on this one (e.g
 # rspec-rails & cucumber-rails). So a deprecation warning is needed if we want
 # to remove it.
```

Comment:
☝🏻 I imagine this applies to `TEST_TYPES` too.

Not to mention the disruption that pulling out half its values could create... or the awfulness of introducing *another* global constant.